plugin-server/src/main/services/schedule.ts

```diff
@@ -77,6 +77,10 @@ export async function startPluginSchedules(
 }
 
 export async function loadPluginSchedule(piscina: Piscina, maxIterations = 2000): Promise<Hub['pluginSchedule']> {
+    await piscina.broadcastTask({ task: 'reloadSchedule' })
```

Comment:
I have no clue whether this is correct or not since `reloadSchedule` assumes certain level of first load done-ness.